### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by El-Potato-Slayer**
-Created active record project which is reddit (micro-reddit)
-Created user, post, and comment models.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :user
+
+  validates :post, presence: true
+end
```
Comment (lekegitrepo) [inline app/models/post.rb]: - **_There should be validations for `body` and `title` in the Post model ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/postValidation.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails)). This implies that you need to add the columns for the title and body (you can change the post column to body or not) to the post table_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
#### Hi @El-Potato-Slayer and @Lidya1234 :clap: :clap:
#### Well done :muscle: The project is well implemented. However, you need to fix the following :point_down: issue before this project can be approved.

**_Submit for another review after implementing the required changes_**

#### _Reviewed by Temitayo Adeleke_

File: README.md [hunk 2]
```diff
@@ -1,17 +1,63 @@
...
+### Prerequisites
+
+To get this project up and running locally, you must already have ruby and necessary gems installed on your computer
+
+**To get this project set up on your local machine, follow these simple steps:**
+
+1. Open Terminal.
+2. Navigate to your desired location to download the contents of this repository.
+3. Copy and paste the following code into the Terminal: git clone https://github.com/El-Potato-Slayer/active-record
+4. Run ```active_record```.
```
Comment (acushlakoncept) [inline README.md]: Did you mean `cd active_record` here? if so kindly update the readme file to update that.
Also `yarn install` should be part of the list. I couldn't proceed without running that command.

**Review by acushlakoncept — APPROVED** [review]
# Status: PROJECT APPROVED  🎉 🚀

### Hi, Team 👋    
I see you have successfully the changes requested by the previous TSE. 

Just before you merge, make sure to update the few changes below and the comments I made on your readme file.

Update your PR title to a meaningful one:  Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Pull request merged**